StripeFinancialConnections/StripeFinancialConnections/Source/Analytics/FinancialConnectionsAnalyticsClient.swift

```diff
@@ -45,17 +45,12 @@ final class FinancialConnectionsAnalyticsClient {
     public func logExposure(
         experimentName: String,
         assignmentEventId: String,
-        parameters: [String: Any] = [:]
+        accountholderToken: String
     ) {
-        var parameters = parameters.merging(
-            additionalParameters,
-            uniquingKeysWith: { eventParameter, _ in
-                // prioritize event `parameters` over `additionalParameters`
-                return eventParameter
-            }
-        )
+        var parameters = additionalParameters
         parameters["experiment_retrieved"] = experimentName
         parameters["arb_id"] = assignmentEventId
+        parameters["account_holder_id"] = accountholderToken
```

Comment:
the code before:

```
parameters: ["account_holder_id": accountHolder])
```

Reply:
and the additional params are gonna be pulled in .log(... function essentially?

Reply:
@vardges-stripe if I understood correctly, IF there is a need for future parameters, then we can add a `parameters` field

however, right now, from the Slack discussions, all these parameters are currently required (so they have explicit parameters vs. a generic dictionary)